packages/mobx-react-lite/__tests__/strictAndConcurrentMode.test.tsx

```diff
@@ -42,36 +41,32 @@ test("uncommitted observing components should not attempt state changes", () =>
     }
 })
 
-const strictModeValues = [true, false]
+const strictModeValues = [true]
```

Comment:
If checking in non-strict mode is no longer necessary, can we get rid of `.forEach` loop here?